models/config_item.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+// Config Types
+const (
```

Comment:
I don't think these shold be global - maybe just have them in the dummy pacakge - Also don't use acronoym i.e. ConfigTypeNode